README.md

```diff
@@ -144,3 +144,36 @@ The workflow using `setNextRequest` allows us to control order of tests, to do t
 1. Getting a product by previously stored id. 
    
 For an example of tests, check out `products` and `product` within the Postman Collection.
+
+## Build & Deploy
+
+To build the server:
+
+1. Run `yarn build` and you'll have output in the `build/dist/` directory
+2. Run `yarn start` to run the server locally from that built output
+
+Deploying the server can be done in various ways. Checkout the [Apollo Documentation on deployment](https://www.apollographql.com/docs/apollo-server/deployment/) for suggestions on how to deploy it several cloud hosting services.
+
+Alternatively, you can copy the build output and host it yourself in a dedicated machine with NodeJS installed.
+
+Another alternative is to deploy in a kubernetes cluster by building a docker image:
+
+```dockerfile
```

Comment:
Just a suggestion, it might be good if we create a dockerfile for it in the repo also.

Reply:
Yeah, I was contemplating on that too. Decided to skip it for now since don't have plans for "hosting" or "publishing" the docker image.

Reply:
yeah, having the docker file itself that we can reference from here would be handy. but that can happen later too.